src/sheba/mod.rs

```diff
@@ -0,0 +1,42 @@
+pub fn verify_card_number<T: AsRef<str>>(number: T) -> bool {
+    let number = number.as_ref();
+    let len = number.len();
+    if len < 16
+        || number.chars().any(|c| !c.is_numeric())
```

Comment:
i think use `number.parse::<i32>().is_err()` better performance

Reply:
what??? it shouldn't be here :D it's absolutely incomplete!